src/main/scala/za/co/absa/ultet/util/parsers/yaml/DBTableFromYaml.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2023 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.ultet.util.parsers.yaml
+
+import cats.syntax.either._
+import io.circe.{Error, Json, ParsingFailure}
+import io.circe.yaml.{parser => yamlParser}
+import io.circe.generic.extras.ConfiguredJsonCodec
+import za.co.absa.ultet.model.table.TableDef
+import za.co.absa.ultet.types.DatabaseName
+import za.co.absa.ultet.types.schema.SchemaName
+import za.co.absa.ultet.types.table.{TableIdentifier, TableName}
+import za.co.absa.ultet.types.user.UserName
+
+@ConfiguredJsonCodec
+case class DBTableFromYaml(
+  table: String,
+  description: Option[String],
+  primaryDb: String,
```

Comment:
why `primary` ?

also, isn't schema missing?

Reply:
Nope. Schema is decided by the folder the source is in - so much easier to mover if needed.

Reply:
> why `primary` ?

For eventual multi-db (cluster) support. I know it's far ahead, but it's a remnant from the original design.